tutorials/spectral-cube-reprojection/SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>Given the redshift of this observation, we expect the first cube to cover...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>There's nothing here related to redshift.  These are different observed frequencies and the lines are being observed at (approximately) their rest frequency.  I don't think additional clarification here helps much.  The first line describes the observations, the second two lines describe what we're extracting from them.  This is like taking a spectrum centered around 6600 angstroms and 4900 angstroms, then noting that the former covers H-alpha and the latter covers H-beta.</p>

Reply:
<p>Got it. What I am trying to do is to differentiate between information which you know because you took the data and have experience and information which you can figure out by looking at the data. So lemme try this language:</p><p>The data in the cubes was observed at two frequencies, 139 and 89 GHz. We're expecting the 139 Hz data set to include an H2CS emission line (4(1,3)-3(1,2) transition with a rest wavelength 139.483699) and the 89 GHz data to include an SiO emission line (v=5-4 transition with a rest wavelength of 217.1049).</p><p> </p><p>QUESTION: How does the 89 GHz data include a line at 217 GHz?  </p>

Reply:
Got it, can put in something like that.

To answer your question: by the power of typos!  Good catch